### ISSUE TITLE
RE: Whisper to ChatGPT

### ISSUE THREAD

**Issue by ericdrgn** [bramenn/whisper-matrix-plugin#1]
https://github.com/matrixgpt/matrix-chatgpt-bot/issues/62

Since you already have the matrix whisper plugin how hard would it be to introduce the official whisper API and have it call the matrix chatgpt bot? I think the feature is useful as well but I'm unsure how to add all that is needed to your current plugin.

**Comment by bramenn** [comment]
@ericdrgn That could easily be done with the API, since it is not tied to any programming language. The detail is that maybe it is not in the coverage of the gpt bot chat project, it would be best to talk to the developers of that project.

**Comment by bramenn** [comment]
I would have no problem to make this plugin use the API, we would just have to pay for it.

**Comment by ericdrgn** [comment]
Yeah I think everyone is cool with that at this point, if we can get a bot into matrix that uses the whisper api that would be awesome. I don't really know how that would change your code but if you think it would be easy we should go for it. That would allow you to do the thing you requsted for the matrix-chatgpt-bot
